### PR TITLE
RCPS-144 Student Information Management: Updating student repeating email address that already exist

### PR DESCRIPTION
**Pull request by kylmrcdr123**
The user can’t use an email address twice or that is already being used inside the system/database.

### CODE REVIEW

File: src/main/java/com/student/information/management/appl/facade/student/StudentFacade.java [hunk 1]
```diff
@@ -38,4 +38,6 @@ public interface StudentFacade {
      * @return true if update is successful.
      * */
     boolean updateStudent(Student student);
+
+    Student findStudentByEmail(String email);
```
Comment (RoCS2024) [inline src/main/java/com/student/information/management/appl/facade/student/StudentFacade.java:42]: add documentation

File: src/main/java/com/student/information/management/data/connection/ConnectionHelper.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
  */
 public class ConnectionHelper {
     /** The connection URL. */
-    public static final String URL = "jdbc:oracle:thin:@localhost:1521:oracleDB";
+    public static final String URL = "jdbc:oracle:thin:@localhost:1521:rogate";
```
Comment (RoCS2024) [inline src/main/java/com/student/information/management/data/connection/ConnectionHelper.java:15]: do not change the ssid

File: src/main/java/com/student/information/management/data/student/dao/StudentDao.java [hunk 3]
```diff
@@ -47,4 +47,5 @@ public interface StudentDao {
      * */
     boolean updateStudent(Student student) ;
 
+    Student findStudentByEmail(String email);
```
Comment (RoCS2024) [inline src/main/java/com/student/information/management/data/student/dao/StudentDao.java:50]: add documentation

**Review by RoCS2024 — CHANGES_REQUESTED** [review]
see comments

File: src/main/java/com/student/information/management/data/student/dao/StudentDao.java (hunk 3 above)
Comment (kylmrcdr123) [reply src/main/java/com/student/information/management/data/student/dao/StudentDao.java:56]: noted po

**Review by RoCS2024 — APPROVED** [review]
all good

**Pull request merged**